release.config.js

```diff
@@ -35,6 +36,12 @@ module.exports = {
         ],
       },
     ],
+    [
+      '@semantic-release/exec',
+      {
+        publishCmd: 'git checkout -- package.json',
+      },
+    ],
```

Comment:
This part, to not update the package.json version is required only when using mono-repo. 

If we don't want to update the version, maybe put `0.0.0` in `package.json`

Reply:
I think I will remove `@semantic-release/exec` instead since this is not a mono repo.